Dockerfile

```diff
@@ -57,9 +64,6 @@ RUN gem install circle-cli
 # Make sure we are on the latest version of Composer
 RUN composer selfupdate
```

Comment:
`composer selfupdate --2` would be good here to lock it in to at least a major version.